connd/qconnectionagent.cpp

```diff
@@ -30,6 +30,8 @@
 #define CONND_PATH "/Connectiond"
 #define CONND_SESSION_PATH = "/ConnectionSession"
 
+Q_LOGGING_CATEGORY(connAgent, "org.nemomobile.connectionagent", QtInfoMsg)
```

Comment:
```suggestion
Q_LOGGING_CATEGORY(connAgent, "org.sailfishos.connectionagent", QtWarningMsg)
```

Use sailfishos instead and QtWarningMsg as default to reduce default logging.

Reply:
Done